docs/installation/basic.md

```diff
@@ -63,18 +145,23 @@ python3 create_db.py
 # enter email and password
 python3 manage.py db stamp head
 ```
+**Note 1:** In case you made your own username and password in Step 2 are now getting `FATAL:  password authentication failed for user "john"` , probable cause is non updation of `.env` file. To resolve it, open the `.env` file and update `DATABASE_URL=postgresql://USERNAME:PASSWORD@127.0.0.1:5432/oevent` and you are good to go.
 
+**Note 2:** In case you are using Anaconda distribution for python, you may get an import error regarding `celery.signals` module. Please use the default python version while executing these steps in that case.
 
 * **Step 6** - Start the application along with the needed services.
 The `&` at the end of the commands below make them run in background so that they don't hold the terminal.
 
 ```sh
-# download and run redis
-wget http://download.redis.io/releases/redis-3.2.1.tar.gz
-tar xzf redis-3.2.1.tar.gz
-rm redis-3.2.1.tar.gz
-cd redis-3.2.1
-make
+# Install and run redis
+# For Ubuntu, Debian and alike
+sudo apt-get install redis-server
+# For Fedora, RedHat, CentOS
+sudo dnf install redis
+
+# For macOS
+brew install redis
+brew services start redis
 
 # To run redis
 redis-3.2.1/src/redis-server &
```

Comment:
This step isn't needed since redis above is installed using apt

Reply:
Done